src/utils/mutations/SetUserSearchEngineMutation.js

```diff
@@ -0,0 +1,24 @@
+import { graphql } from 'react-relay'
+import callMutation from 'src/utils/mutations/callMutation'
+
+const mutation = graphql`
+  mutation SetUserSearchEngineMutation($input: SetUserSearchEngineInput!) {
+    setUserSearchEngine(input: $input) {
+      user {
+        userId
```

Comment:
This will want to fetch the new search engine value. In mutations, the fetched info should include anything that could be modified as a result of the mutation.

Reply:
Probably not a great answer, but we don't have these fields on the UserType in schema.js, that's in another ticket. Can I add a `todo` here (and in the other mutation) to change what fields these classes return?